integreat_cms/locale/de/LC_MESSAGES/django.po

```diff
@@ -3603,12 +3603,16 @@ msgid "Please check the box if you want to use short urls."
 msgstr "Kreuzen Sie an, wenn Sie Kurz-URLs benutzen wollen."
 
 #: cms/models/regions/region.py
-msgid "Enable tunews"
-msgstr "tünews aktivieren"
+msgid "Enable external news"
+msgstr "Externe News aktivieren"
 
 #: cms/models/regions/region.py
-msgid "Enable to show a feed of tunews articles to users."
-msgstr "Auswählen, um anderen Benutzer:innen einen tünews-Feed anzuzeigen."
+msgid ""
+"Enable to display national news from external providers in addition to local "
```

Comment:
I would rephrase that a little bit as we are not sure if we stick with national news and i would differentiate a little bit between push notifications and external news.
```suggestion
"Enable to display external articles in addition to local news managed by the CMS "
```